Bottom: 3251f0ff72f38868bdd0f1adaeb389188c324690
Top:    66a70ed87ef1d97eafc3e2f9b1870e4470074d0d
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 21:27:52 -0700

fixup0-update-scriptinterpreterpython


---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 00af5590dbf..b582f583e23 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -447,9 +447,8 @@ class FileHandleTestCase(lldbtest.TestBase):
             self.assertTrue(re.search(r'error:.*lolwut', errors))
             self.assertTrue(re.search(r'zork', errors))
 
-    #FIXME This shouldn't fail for python2 either.
+
     @add_test_categories(['pyapi'])
-    @skipIf(py_version=['<', (3,)])
     def test_replace_stdout(self):
         f = io.StringIO()
         with replace_stdout(f):
